include/curl/curl.h

```diff
@@ -1726,6 +1726,7 @@ enum {
   CURL_HTTP_VERSION_1_0,  /* please use HTTP 1.0 in the request */
   CURL_HTTP_VERSION_1_1,  /* please use HTTP 1.1 in the request */
   CURL_HTTP_VERSION_2_0,  /* please use HTTP 2 in the request */
+  CURL_HTTP_VERSION_2_0_CLEAN_TCP,  /* please use HTTP 2 over clean TCP */
```

Comment:
I agree the name is odd. I already change it to no_upgrade. 
I actually don't like it either, but I cannot found a good name to specify this feature.
By just looking at HTTP_VERSION_2_0_NO_UPGRADE it is not clear what's happening.